test/functional/helpers/constants/general.js

```diff
@@ -1,3 +1,4 @@
 export default {
-  ecidRegex: /^[0-9]{38}$/
+  ecidRegex: /^[0-9]{38}$/,
+  nonce: "123"
 };
```

Comment:
I know this module existed before, but I would advise we move away from `general.js` and just create a file for each constant. Alternatively, we keep general.js but export each constant as its own named export:

```
export const ECID_REGEX = ...
export const NONCE = ...
```

Anyway, don't let it hold up the merge.

Reply:
ok, I will add a ticket for introducing a nonce in the functional tests that will make it part of the process and also populate the alloy code snippet like we discussed earlier.